fix(curriculum): remove explanation for textContent property

Vijaychandra-Govindapalle opened the pull request:
Removes explanation for textContent property in step 14 of  date formatter project in  javascript algorithms and datastructures(beta)  course. As it's explanation already exists in music player project

Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ x] My pull request targets the `main` branch of freeCodeCamp.
- [ x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #53122

<!-- Feel free to add any additional description of changes below this line -->

Vijaychandra-Govindapalle committed:
fix: remove explanation for textContent property

Removes explanation for textContent property in step 14 of  date formatter project
in  javascript algorithms and datastructures(beta)  course. As it's explanation already
exists in music player project

Commit:
feat: add link to Discord (#53129)

pratyansh-agrawal committed:
fix(curriculum): calorie counter step 94 ensure function is passed (#53112)

Co-authored-by: Krzysztof G. <60067306+gikf@users.noreply.github.com>

Vijaychandra-Govindapalle committed:
Merge branch 'main' into fix/two-instances-of-explanation-for-the-textContent-property

gikf reviewed (commented):
From my understanding the example also should be removed.
````
```html
<p id="example-paragraph">Example Text</p>
```
```js
const exampleParagraph = document.getElementById("example-paragraph");
console.log(exampleParagraph.textContent); // "Example Text"
exampleParagraph.textContent = "New Text";
console.log(exampleParagraph.textContent); // "New Text"
```
````

shivam-0510 committed:
fix(curriculum): correct form button text in Todo App (#53094)

Co-authored-by: jdwilkin4 <jwilkin4@hotmail.com>
Co-authored-by: Jessica Wilkins <67210629+jdwilkin4@users.noreply.github.com>
Co-authored-by: Huyen Nguyen <25715018+huyenltnguyen@users.noreply.github.com>

Vijaychandra-Govindapalle committed:
Merge branch 'freeCodeCamp:main' into fix/two-instances-of-explanation-for-the-textContent-property

Vijaychandra-Govindapalle committed:
fix: remove explanation for textContent property
Removes explanation of textContent Property in step 14 of date formatter project
in javascript algorithms and datastructures(beta) course. As explanation for the same
exists in step 56 of music player project in the same course.

Vijaychandra-Govindapalle committed:
Merge branch 'fix/two-instances-of-explanation-for-the-textContent-property' of https://github.com/Vijaychandra-Govindapalle/freeCodeCamp into fix/two-instances-of-explanation-for-the-textContent-property
